src/app/components/spartan/ui-input-helm/src/lib/hlm-input.directive.ts

```diff
@@ -4,7 +4,7 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import type { ClassValue } from 'clsx';
 
 export const inputVariants = cva(
-	'tw-flex tw-box-border tw-rounded-[10px] tw-border tw-font-normal tw-border-input tw-bg-transparent tw-text-sm  file:tw-border-0 file:tw-text-foreground file:tw-bg-transparent file:tw-text-sm file:tw-font-medium placeholder:tw-text-muted-foreground focus-visible:tw-outline-none focus-visible:tw-ring-2 focus-visible:tw-ring-purple focus-visible:tw-ring-offset-2 disabled:tw-cursor-not-allowed disabled:tw-opacity-50',
+	'tw-flex tw-box-border tw-rounded-[10px] tw-border tw-font-normal  tw-bg-transparent tw-text-sm  file:tw-border-0 file:tw-text-foreground file:tw-bg-transparent file:tw-text-sm file:tw-font-medium placeholder:tw-text-muted-foreground focus-visible:tw-outline-none focus-visible:tw-ring-2 focus-visible:tw-ring-purple focus-visible:tw-ring-offset-2 disabled:tw-cursor-not-allowed disabled:tw-opacity-50',
```

Comment:
this change breaks buttons on other pages. e.g. login page.

Reply:
ok weird i was under the impression that all inputs use <oeb-input> for styling ..

Reply:
just found out that you can prioritize classes in tw with `!tw-bg-transparent`